src/parser/jobs/smn/modules/Aetherflow.js

```diff
@@ -88,10 +94,16 @@ export default class Aetherflow extends Module {
 		this._badDotReqCasts[actionId][statusesMissing]++
 	}
 
-	_onPainflareDamage(event) {
-		// Only fault single target PFs _outside_ rushes.
-		if (event.hits.length <= 1 && !this.gauge.isRushing()) {
-			this._badPainflares.push(event)
+	_onAoeDamage(event) {
+		if (event.ability.guid === ACTIONS.PAINFLARE.id) {
```

Comment:
would be good to move the breakpoints into config up the top, potentially. having separate conditionals for each is a bit ehh.

if that's not feasible, can you move the nested conditionals out to the outer statements pl0x